Hymns/Display/HymnLyricsView.swift

```diff
@@ -11,20 +11,28 @@ public struct HymnLyricsView: View {
     
     public var body: some View {
         guard let lyrics = viewModel.lyrics else {
-            return Text("error!")
+            return AnyView(Text("error!"))
         }
         
         guard !lyrics.isEmpty else {
-            return Text("loading...")
+            return AnyView(Text("loading..."))
         }
-
-        return Text(lyrics[0].verseContent[0])
+        
+        return AnyView(
+            ForEach(lyrics, id: \.self) { verse in
+                VStack(alignment: .leading) {
+                    Spacer()
+                    ForEach(verse.verseContent, id: \.self) { line in Text(line)
+                    }
+                }
+            }
+        )
```

Comment:
This looked a little funky when I built it, especially in the Previews. For some reach each verse was its own screen.

Also, if you put the spacer on top, that means short songs will all get smushed to the bottom. Would something like this work:

```suggestion
        
        return AnyView(
            VStack(alignment: .leading) {
                ForEach(lyrics, id: \.self) { verse in
                    Group {
                        ForEach(verse.verseContent, id: \.self) { line in
                            Text(line)
                        }
                    }
                }
            }
        )
```

Then after doing that, all you need to do is fix the formatting to look like the figma spec

Reply:
Nice! I added a further commit expanding off of what you did here. I think the spacing is starting to look good. Also, I tested with changing the accessibility to large system font etc. and everything seems to come out looking pretty good.  

One question about the figma spec is that Apple really seems to be promoting dynamic type which is in SwiftUI by default that changes their font size with the system. If we lock in a font size and spacing to a specific font size like 16pt for example then the lyrics will no longer adjust with the system. I was thinking we could still have options like your other app but the options would only change dynamic fonts from like normal body to large etc... Idk what you think?